packages/output-view/README.md

```diff
@@ -0,0 +1,6 @@
+# Theia - Output View Extension
```

Comment:
Can we name it `output` not `output-view`, as it sounds too limited to just ui.
I think it should also contain the manager (see my comment above) and longer term even a stub on the backend, so the backend process can broadcast output messages to the frontends.

Reply:
Done